Домашка

IIkhsan opened the pull request.

IIkhsan committed:
Домашка

ZalyalovIldar reviewed (changes requested):
на первом сценираа в сториборде жесть какая - то, и на 2-ом еще раз посмотри констрэйнты, многие лишние, повторяются, в итоге например первый сценарий поплыл на iPhone 7+

IIkhsan committed:
Сделал

на первом сценирии в сториборде исправил, и на 2-ом убрал лишние
констрейнты(если убирал что-то еще все падало),  в итоге первый
сценарий не поплыл на iPhone 7+

IIkhsan commented:
на первом сценирии в сториборде исправил, и на 2-ом убрал лишние констрейнты(если убирал что-то еще все падало),  в итоге первый сценарий не поплыл на iPhone 7+
